meson.py

```diff
@@ -14,13 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from mesonbuild import mesonmain
-import sys, os
+import sys
+from pathlib import Path
+
+# If we're run uninstalled, add current directory to sys.path to ensure that we
```

Comment:
Technically this is not current directory (which makes me think "current working directory" or ".", which is already always in `sys.path`), but the parent of the script. I think the comment should be a bit clearer about that.